build-system/check-package-manager.js

```diff
@@ -122,8 +123,8 @@ function performNodeVersionCheck(latestLtsVersion) {
 
 // If yarn is being run, perform a version check and proceed with the install.
 function performYarnVersionCheck() {
-  const yarnVersion = getStdout('yarn --version').trim();
-  const yarnInfo = getStdout('yarn info --json yarn').trim();
+  const yarnVersion = getStdout(yarnExecutable + ' --version').trim();
```

Comment:
Globally installed packages are unnecessary. Why require 2 steps (`npm install -g yarn gulp`, `gulp build`) when it can just be `npx gulp build`?

Reply:
I think @kristoferbaxter was specifically asking about `yarn`, not `gulp`.

But yeah, once this PR is ready, it will ensure that local development will work without global packages.